client/src/app/services/authorization.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { UserFacade } from 'store';
+import { User } from 'models';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthorizationService {
+
+  private user$;
```

Comment:
please specify interface, $ in the end points to stream, as I can see this variable is not a stream